### DISCUSSION TITLE
Hardcoded .container width

### DISCUSSION THREAD

**Discussion by cernyjakub** [bootstrap-email/bootstrap-email#180] [Ideas]
Hello!

`.container` width is hardcoded in `core/scss/components/_container.scss`

It would be perfect to have it as a SASS variable, there are some cases when changing this makes sense.

Thanks for a nice tool!
jakub

**Comment by stuyam** [comment]
Yeah that's not a bad idea, it is hard coded because it is based on a best practice of the best support in most email clients. However there are cases where I can see people wanting to modify this.
